src/services/globals.ts

```diff
@@ -1,4 +1,4 @@
-const TEST_MODE = false;
+const TEST_MODE = true;
```

Comment:
We should probably change this before merging?

Reply:
Created an issue for it 😄  #138